src/utils/index.js

```diff
@@ -294,15 +294,52 @@ export const doesPlotContainCrop = plot =>
 
 export const getLifeStageRange = memoize((
   /** @type {farmhand.cropTimetable} */ cropTimetable
-) =>
-  [SEED, GROWING].reduce(
+) => {
+  return [SEED, GROWING].reduce(
```

Comment:
Thanks for making this change. There are a lot of remnants in this code base of me getting overly clever with arrow function expressions. 😅